app/components/excerpt/excerpt.css

```diff
@@ -0,0 +1,24 @@
+.Excerpt {
+  display: block;
+}
+
+.Excerpt-header {
+  display: flex;
+}
+
+.Excerpt-title {
+  flex-shrink: 1;
+  flex-grow: 0;
+}
+
+a.Excerpt:hover .Excerpt-title {
+  text-decoration: underline;
+}
+
+.Excerpt-date {
+  flex-shrink: 0;
+  flex-grow: 0;
+  font-size: var(--font-size-s);
+  color: var(--color-text-light);
+  margin-left: var(--spacing-unit-s);
+}
```

Comment:
@roschaefer: I’m unsure how this component might be called. In this PR, it’s only used to display a single reply to a request. But it might as well be used to display a list of requests a user has replied to etc. Basically anything that has a title, some kind of text and an optional date.

Naming suggestions welcome! :)

![Screenshot 2020-05-19 17 37 54](https://user-images.githubusercontent.com/1512805/82347382-cfc53080-99f7-11ea-8986-081687083ab1.png)

Reply:
Plädiere für „Antwort“
Im klassischen CRM ein „Kontakt“

Reply:
Ja, aber das ist mehr eine generische Komponente, die einen (verlinkten) Titel, einen kurzen Text und optionale Meta-Informationen wie z.B. eine Zeit darstellt. Könnte wie gesagt auch verwendet werden für:

* eine Liste von Recherchen
* eine Liste von Recherchefragen
* eine Liste von Lokalexpert*innen, sortiert nach Aktivität
* …

Reply:
Meinst du das rot-umrandete? 
"Informationsblock"?
"Informationseinheit"?

Reply:
Can't we use our message component here?

`Excerpt` as a css class name is fine for me.